.github/workflows/add-artifacts-to-release.yml

```diff
@@ -0,0 +1,88 @@
+name: Release artifacts
+on:
+  release:
+    types: [published]
+
+jobs:
+  publish_libcorerust:
+    name: Build and upload libcorerust
+    runs-on: ubuntu-latest
+    permissions: write-all
+    steps:
+      - uses: actions/checkout@755da8c3cf115ac066823e79a1e1788f8940201b
+        with:
+          ref: ${{ github.event.release.tag_name }}
+          fetch-depth: 0
+      - uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
+      - name: Set up JDK 17
+        uses: actions/setup-java@v2
+        with:
+          distribution: "zulu"
+          java-version: "17"
+      - name: Cache Gradle packages
+        uses: actions/cache@v1
+        with:
+          path: ~/.gradle/caches
+          key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle') }}
+          restore-keys: ${{ runner.os }}-gradle
+      - name: Run local core-rust docker build
+        run: ./gradlew :core-rust:buildRustForDocker -P rustBinaryBuildType="release"
+      - name: Upload libcorerust
+        run:
+          gh release upload ${{ github.event.release.tag_name }} --repo radixdlt/babylon-node ${{ github.workspace }}/core-rust/target/x86_64-unknown-linux-gnu/release/libcorerust.so --clobber
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
We may need to pull this from AWS secrets

Reply:
its the standard GITHUB_TOKEN. Its always available on hosted runners. We actually cant put this into AWS.